### PR TITLE
Moved the Blazor Accessibility sample app to .NET 5 RTM

### PR DESCRIPTION
**Pull request by mkArtakMSFT**

### CODE REVIEW

File: samples/aspnetcore/blazor/Accessibility/Accessibility.csproj [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
     <TargetFramework>net5.0</TargetFramework>
-    <RuntimeIdentifier>browser-wasm</RuntimeIdentifier>
-    <UseBlazorWebAssembly>true</UseBlazorWebAssembly>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" Version="5.0.0-preview.7.20365.19" />
-    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.DevServer" Version="5.0.0-preview.7.20365.19" PrivateAssets="all" />
-    <PackageReference Include="System.Net.Http.Json" Version="5.0.0-preview.7.20364.11" />
+    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" Version="5.0.1" />
```
Comment (captainsafia) [inline samples/aspnetcore/blazor/Accessibility/Accessibility.csproj:8]:
```suggestion
    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" Version="5.0.2" />
```

Unless there's a reason we're setting to `5.0.1` we might as well target `5.0.2`.

**Review by captainsafia — APPROVED** [review]

File: samples/aspnetcore/blazor/Accessibility/Accessibility.csproj [hunk 2]
```diff
@@ -1,15 +1,13 @@
...
-    <RuntimeIdentifier>browser-wasm</RuntimeIdentifier>
-    <UseBlazorWebAssembly>true</UseBlazorWebAssembly>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" Version="5.0.0-preview.7.20365.19" />
-    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.DevServer" Version="5.0.0-preview.7.20365.19" PrivateAssets="all" />
-    <PackageReference Include="System.Net.Http.Json" Version="5.0.0-preview.7.20364.11" />
+    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" Version="5.0.2" />
+    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.DevServer" Version="5.0.1" PrivateAssets="all" />
```
Comment (captainsafia) [inline samples/aspnetcore/blazor/Accessibility/Accessibility.csproj:9]: @mkArtakMSFT This package should be updated to 5.0.2 as well.

Comment (mkArtakMSFT) [reply samples/aspnetcore/blazor/Accessibility/Accessibility.csproj:9]: Will update these in a separate PR. Thanks!

Comment (mkArtakMSFT) [reply samples/aspnetcore/blazor/Accessibility/Accessibility.csproj:9]: Thanks Safia: https://github.com/aspnet/samples/pull/68/files

**Comment by SteveSandersonMS** [comment]
Very nice

**Pull request merged**